components/feature/tabs/src/main/java/mozilla/components/feature/tabs/WindowFeature.kt

```diff
@@ -46,14 +46,12 @@ class WindowFeature(
                         WindowRequest.Type.OPEN -> consumeWindowRequest(state.id) {
                             if (state.content.private) {
                                 tabsUseCases.addPrivateTab(
-                                    url = windowRequest.url,
```

Comment:
Only consequence of this is that we'll see an initial "about:blank" load for new windows (target="_blank" links and window.open). 

We have a workaround for this in `GeckoEngineSession`: https://github.com/mozilla-mobile/android-components/blob/master/components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt#L436-L441 but it won't apply here. 

Since this behaved the same in Fennec though we decided it's fine to land this.